transactions/scripts/get_nft_metadata.cdc

```diff
@@ -22,28 +28,44 @@ pub struct NFT {
         owner: Address,
         nftType: String,
         royalties: [MetadataViews.Royalty],
+        externalURL: String,
         collectionPublicPath: PublicPath,
         collectionStoragePath: StoragePath,
         collectionProviderPath: PrivatePath,
         collectionPublic: String,
         collectionPublicLinkedType: String,
-        collectionProviderLinkedType: String
+        collectionProviderLinkedType: String,
+        collectionName: String,
+        collectionDescription: String,
+        collectionExternalURL: String,
+        collectionSquareImage: String,
+        collectionBannerImage: String
     ) {
         self.name = name
         self.description = description
         self.thumbnail = thumbnail
         self.owner = owner
         self.type = nftType
         self.royalties = royalties
+        self.externalURL = externalURL
         self.collectionPublicPath = collectionPublicPath
         self.collectionStoragePath = collectionStoragePath
         self.collectionProviderPath = collectionProviderPath
         self.collectionPublic = collectionPublic
         self.collectionPublicLinkedType = collectionPublicLinkedType
         self.collectionProviderLinkedType = collectionProviderLinkedType
+        self.collectionName = collectionName
+        self.collectionDescription = collectionDescription
+        self.collectionExternalURL = collectionExternalURL
+        self.collectionSquareImage = collectionSquareImage
+        self.collectionBannerImage = collectionBannerImage
     }
 }
 
+pub struct NFTCollection {
+    
+}
```

Comment:
What is this?

Reply:
My mistake - was thinking of reformatting things, and left that over. Removing it.